about.md

```diff
@@ -2,29 +2,37 @@
+npm, Inc. is a company co-founded in 2014 by npm’s creator, [Isaac Z. Schlueter](https://github.com/isaacs), and [Laurie Voss](https://github.com/seldo).
 
 npm, Inc. is dedicated to the long term success of the JavaScript community, which includes the success of the open-source Node.js and npm projects.
 
 At npm, Inc. we do three things to support this goal:
 
-1. Run the open source registry as a free service.
-2. Build tools and operate services that support the secure use of packages in a private or enterprise context.
-3. Build innovative new tools and services for the developer community.
+1. Run the open source npm Registry as a free service.
+2. Build tools and operate services that support securely using packages in a private or enterprise context.
+3. Build innovative new tools and services to reduce developer friction.
 
-npm's mission is to take Open Source development to entirely new places. When everyone else is adding force, we work to reduce friction.
+npm’s mission is to take Open Source development to entirely new places. When everyone else is adding force, we work to reduce friction.
 
-npm is not a typical product, and we are not a typical "work hard/play hard" startup. We are responsible adults with diverse backgrounds and interests, who take our careers and our lives seriously. We believe that the best way to iterate towards success is by taking care of ourselves, our families, our users, and one another. We aim for a sustainable approach to work and life, because that is the best way to maximize long-term speed, while retaining clarity of vision. Compassion is our strategy.
+Our technology has been embraced by millions of developers worldwide for client- and server-side applications as diverse as IoT, mobile development, financial services, and aerospace. More than 30,000 companies, including the BBC, DocuSign, IBM, SiriusXM, Slack, and Visa, rely on npm’s products and services to reduce developer friction and build amazing things.
 
-Our offices are in downtown Oakland, California.
+npm is not a typical product, and we are not a typical early-stage “work hard/play hard” startup. We are responsible adults with diverse backgrounds and interests, who take our careers and our lives seriously. We believe that __the best way to iterate towards success is by taking care of ourselves, our families, our users, and one another.__
+
+We aim for a sustainable approach to work and life, because that is the best way to maximize long-term speed, while retaining clarity of vision. Compassion is our strategy.
+
+Our offices are in Oakland, California.
 
 ## Contacting npm
 
-You can reach us for support or any other questions via our [contact form](/support) or at [npm@npmjs.com](mailto:npm@npmjs.com).
+You can reach us for __support__ via our [contact form](/support) or at [support@npmjs.com](mailto:support@npmjs.com).
```

Comment:
should we just link to the npm contact page? i find that the redundancy of contact  info across the site makes it hard to  know who to contact